portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,30 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function addRandomQuote() {
+  const quoteContainer = document.getElementById('quote');
+  const authorContainer = document.getElementById('author')
+  fetch("https://type.fit/api/quotes")
+    .then(response => {
+      return response.json();
+    })
```

Comment:
Can use short form of anonymous functions when it is a single statement and the value is returned.

```javascript
.then(response => response.json())
```

Reply:
Changes made